### PR TITLE
Some typos, tweaks, and minor error corrections in ES translation

### PR DESCRIPTION
**Pull request by mberasategi**

### CODE REVIEW

**Comment by brianamarie** [comment]
Awesome, thank you for this, @mberasategi! 

sparkley gif

@davenowell @droidpl would you have bandwidth to review this PR?

**Review by davenowell — CHANGES_REQUESTED** [review]
@mberasategi - I have requested one change.  Thank you for your work!

File: responses/es/04_commit-something.md [hunk 1]
```diff
@@ -10,21 +10,21 @@ Crear una rama te permite hacer modificaciones a tu proyecto sin tener que cambi
...
 
-En el primer campo, escribe un mensaje de confirmación . El mensaje debe informar a tus colaboradores de forma breve sobre los cambios que has introducido al archivo. 
+En el primer campo, escribe un mensaje de confirmación (o _commit message_). El mensaje debe informar a tus colaboradores de forma breve sobre los cambios que has introducido al archivo. 
 
 ### Recomendaciones para los mensajes de confirmación:
 
 - No termines tu mensaje con un punto. 
 - Mantén tu mensaje con 50 o menos caracteres. Si es necesario añade detalles extra en la ventana de descripción extendida. Esta se encuentra justo debajo de la línea de asunto. 
-- Usa la voz activa. Por ejemplo, "añade" en vez de "añadí" y "une" en vez de "uní" 
+- Usa la voz activa. Por ejemplo, "añade" en vez de "añadí" y "une" en vez de "uní".
```
[the remark this answers is not in the document]
Comment (mberasategi) [reply responses/es/04_commit-something.md:19]: Agree. I didn't even consider this. Will change to:

> - Usa la voz activa. Por ejemplo, "Añade la funcionalidad xx" en vez de "Añadida la funcionalidad xx" y "Combina los cambios" en lugar de "Combinados los cambios".

**Comment by davenowell** [comment]
> BTW @davenowell , here's where we have the `propuesta` (for issue) all over the place. It should be changed to `incidencia`, right? Will do as soon as you confirm!

@mberasategi  If you could refer to http://docs.github.com/es and use whichever word appears there, that would be fine. I think the word "propuesta" is used.

**Comment by mberasategi** [comment]
Alright, Docs says "propuesta". I've went through [replacing all "propuesta"s with "incidencia" in Communicating using Markdown](https://github.com/githubtraining/communicating-using-markdown/pull/28#discussion_r483594142)... 🤦🏼  

I guess those should be replaced back then!

**Pull request merged**